### DISCUSSION TITLE
Staining strategy and cycles

### DISCUSSION THREAD

**Discussion by sergilopezserrano** [IBEXImagingCommunity/ibex_imaging_knowledge_base#60] [Q&A] [answered]
Hi all,

I am a postdoctoral researcher in Infection Biology at the Pompeu Fabra University in Barcelona. We want to apply the IBEX technology for our studies and since I started to work with a small testing panel with primary labelled and primary + secondary labelled antibodies (that worked :)). I am seeking guidance on optimizing the staining strategy when using multiple cycles and different biomarkers. 
Since in the panel we are planning to carry out we want to include immune markers with different expression levels, my concern is the risk of possible epitope masking in the sample once applying antibodies after several staining cycles. How this possible situation can be avoided? Do we have to “dedicate” each labelling cycle to similar immune cell subsets? Or in the contrary it may be better to stain the less expressed markers in the first cycles and reserve the more abundantly expressed in the later ones?
Many thanks for your attention and advice, 
Best regards from Barcelona, 

Sergi

**Comment by radtkea** [comment]
Hi Sergi!

Welcome to the community! Barcelona is such a beautiful and vibrant city! I like to do the most touristy thing (hop on hop off bus) whenever I visit a new city. See attached.

First of all, warmest congratulations on your success! Very exciting!

**Epitope masking**
Good that you are thinking about this! We’ve observed a cycle effect on antigenicity in less than 2% of antibodies [[Ref]](https://www.nature.com/articles/s41596-021-00644-9#Sec29). You can test the impact of cycle number on antibody performance by comparing the labeling pattern in iterative (IBEX) labeled slides versus serial tissue sections as we demonstrated in [Figure S5](https://www.pnas.org/doi/10.1073/pnas.2018488117).  

Unfortunately, panel design is an iterative and resource-intensive process ([Figure 3](https://www.nature.com/articles/s41592-021-01316-y)). We are working to change that by publishing [Organ Mapping Antibody Panels](https://humanatlas.io/omap) for human reference atlas construction. 

Here are some general guidelines that we follow:
1.	Antibodies are not stripped with IBEX (Figure S2) [PNAS](https://www.pnas.org/doi/10.1073/pnas.2018488117). Therefore, unconjugated primary antibodies need to go in the first (or earlier cycles) if multiple antibodies from the same host are used to prevent cross-reactivity.
2.	I do “dedicate each labelling cycle to similar immune cell subsets.” We provide several example panels in our original papers that can serve as a template for you. For example, we grouped CD8, CD25, FOXP3, and CD4 together for T cells or CD11c, CD68, MHCII, and SIRPa for myeloid cells.
3.	These are guidelines. Science humbles every day. Brianna and I recently encountered a “bad apple spoiling the bunch” phenomenon with CD1a in the thymus. Placement of this antibody (https://www.novusbio.com/products/cd1a-antibody-o10_nbp2-34697af488) in an early cycle negatively impacted labeling of other T cell markers in later cycles. We removed it from the panel. This is a notable exception. 
4.	I have a “hunch” that CD45 may sterically interfere with other immune cell markers. For this reason, I always put it in the last or penultimate cycle. Scientists can be superstitious.

We could try to construct your antibody panel in the “open” if you feel comfortable sharing:
1.	Desired markers
2.	Imaging channels available on your instrument
3.	Tissue type, e.g., mouse lymph node
4.	Tissue preservation method. Strongly recommend how we prepare our samples with the lower fixative (Steps 3-7) from [here](https://www.nature.com/articles/s41596-021-00644-9#Sec29). The reduced fixative and sucrose exchange yields the best quality (we think) for fixed frozen tissues. 

Have a great start to your day,
A
![Barcelona](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/f06b4f3f-8f04-4422-871e-3c99826e05c6)

**Comment by sergilopezserrano** [comment]
Dear Andrea, 

Thank you so much for your kind response and advice. I am very glad that you liked Barcelona!

At the moment we are going to establish a panel for mouse spleen including high or medium expressed markers using the antibody clones according the list of your publications like CD3, CD4, CD8a, CD11c, CD19 and CD169 and the antibodies we dispose at our lab. But is true that in the future we are planning to include another “tricky markers” like XCR1 and Clec4a4 for dendritic cells (that even they are difficult to detect in normal immunofluorescent procedures) or markers for follicular dendritic cells like mfge8. For this reason, I was asking about epitope masking. 

We are working mainly with frozen-fixed mouse spleens, and we are trying in parallel different fixation approaches including paraformaldehyde with sucrose and zinc-fixative with sucrose to avoid the masking of the epitopes from dendritic cells that become quite elusive. The imaging facility of our Institution provides a Stellaris confocal microscope that has 4 lasers and several channels, and we can analyse our samples by the LasX, Imaris or Fiji software. In fact, the microscope can detect many colours but is also true that in the yellow range between AF488 and AF594 one needs to be very careful choosing the right fluorochromes… I will keep you informed about our progress.

As I talked with my boss Prof. Andreas Meyerhans, whenever you come to Barcelona, we will be so pleased to invite you to visit us at our Institute and even give us a talk in case you want. 

Again many thanks for your attention, 

Let’s stay connected, :)

Sergi

**Comment by radtkea** [comment, marked as answer]
Hi Sergi,

Sincere apologies for my delay! It’s been a busy week but your message is delightful! Thank you so much for your kind words and even kinder invitation from you and Prof. Andreas Meyerhans! Barcelona is a wonderful vacation spot. 

Thank you for sharing your exciting research project! You clearly have the expertise and resilience to study myeloid cells. You’ll need it, haha:)

Happy you can use our publications as a starting point for finding clones and good conjugates. I prefer [CD11c in AF647](https://www.biolegend.com/en-us/products/alexa-fluor-647-anti-mouse-cd11c-antibody-2703) at a high concentration. This is a hamster antibody. So you could always amplify, if needed.

Instead of CD19, I would recommend B220 for the B cells. It is also on pDCs (I believe). [CD169 PE](https://www.biolegend.com/en-us/products/pe-anti-mouse-cd169-siglec-1-antibody-7522) (yes PE) is amazing, especially in the spleen.

Is there a reason why you need mfge8? If you just want to visualize FDCs, then I recommend: CD21/CD35 (Clone 7E9) or CD35 (Clone 8C12). These are generally so bright that you can include in a dim fluorophore (AF532) or a channel with lots of auto (AF488) and save your better channels (AF594, AF647) for tricky markers.

We avoid AF594 and instead use iF594 for our panels because AF594, and similar conjugates, require extended incubation periods to reduce signal. [See here. ](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/fluorescent_probes.html) 

You are so correct! Those are tricky markers. [Dr. Wolfgang Kastenmuller ](https://www.med.uni-wuerzburg.de/en/systemimmunologie/research/leukocyte-dynamics-kamu-lab/) and his group have published a great deal on these cell types/markers. Consider reaching out to Wolfgang for help and guidance. He’s a super kind and brilliant human. Wolfgang was able to determine the following (if I remember correctly): 
1.	Fixation (even mild) destroys the XCR1 epitope. 
2.	He injected 1 ug of purified antibody into live mice 1-2 hours before tissue harvest. Tissues were fixed/prepared using normal procedure (fixation, sucrose) after antibody injection. Then, proceeded with labeling (anti-rat secondary) and direct conjugates.
3.	He was using an antibody from Dr. Richard Kroczek. I assume it is this [one](https://absoluteantibody.com/product/anti-xcr1-marx10/).

Sadly, I don’t know anything about Clec4a4. 

Reporter animals could be very helpful, if you can get access to them. 

I’ve always tried to avoid snap frozen tissues because of poor tissue quality/freezing artifacts. However, I recently found the [Seal n’ Freeze](https://vimeo.com/626915917) box. My colleague, Zhixin, and I obtained nice results for snap frozen tissues with this device. Zhixin has done an exhaustive and careful comparison that we are working to share with everyone. Consider snap freezing tissue, applying XCR1 antibody, and doing a post-fix. Then proceed with the rest of your antibodies validated for these tissues.

Lastly, be careful about the detergent in your blocking/staining buffer. Try with and without and see if you are able to visualize your tricky markers. A handful of markers are sensitive to detergent. We are trying to capture this information in the knowledge-base.

Here's a great [place](http://www.dendritics.net/) for myeloid specific markers, mostly human I believe.

I hope this helps! Best of luck and please keep us posted!
A
